### ISSUE TITLE
[fvt]2.12.2 xcatprobe could not installed on servicenode

### ISSUE THREAD

**Issue by junxiawang** [xcat2/xcat-core#1681]
env:all platform
build:
[root@c910f02c01p07 xcat]# lsxcatd -v
Version 2.12.2 (git commit 79426ad21e602b8a1bf5c02084ffb8c8ca9bf01e, built Thu Aug 11 06:16:05 EDT 2016)
[root@c910f02c01p07 xcat]# 
How to reprpoduce:
```
On servicenode:
[root@c910f02c01p07 xcat]# cd /opt/xcat/
[root@c910f02c01p07 xcat]# ls
bin  lib  README  sbin  share  ws  xcatdsklspost  xcatinfo  xcatinstallpost  xdsh
[root@c910f02c01p07 xcat]# ls -al
total 72
drwxr-xr-x. 8 root root  4096 Aug 11 11:33 .
drwxr-xr-x. 3 root root    17 Aug 11 11:29 ..
drwxr-xr-x  2 root root  8192 Aug 11 11:33 bin
drwxr-xr-x  3 root root    35 Aug 11 11:33 lib
-rw-r--r--  1 root root   308 Aug 11 06:16 README
drwxr-xr-x  2 root root  4096 Aug 11 11:33 sbin
drwxr-xr-x  5 root root    37 Aug 11 06:16 share
drwxr-xr-x  2 root root   151 Aug 11 11:33 ws
-rwxr-xr-x. 1 root root 32790 Aug 11 11:29 xcatdsklspost
-rw-r--r--. 1 root root    53 Aug 11 11:32 xcatinfo
-rwxr-xr-x. 1 root root  5006 Aug 11 11:29 xcatinstallpost
drwxr-xr-x  3 root root    20 Aug 11 11:33 xdsh
[root@c910f02c01p07 xcat]# 

----------------------------> no xcatprobe here
```

**Comment by hu-weihua** [comment]
Fixed by pull request #1701, @junxiawang , could you help to verify this bug? thanks

**Comment by junxiawang** [comment]
verified
